spec/services/provider_interface/reference_status_report_service_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::ReferenceStatusReportService do
+  reference_const = 'Reference'
+  context_const = 'Context'
+  describe '#headers' do
+    it 'returns the correct headers' do
+      service = described_class.new([], context: nil)
```

Comment:
You can move the service to a  subject: https://rspec.info/features/3-12/rspec-core/subject/